app/upgrades/v1/constants.go

```diff
@@ -0,0 +1,29 @@
+package v1
+
+import (
+	"github.com/classic-terra/core/v2/app/upgrades"
+	core "github.com/classic-terra/core/v2/types"
+)
+
+var (
+
+	// UpgradeName defines the set of on-chain upgrade name for the classic terra v1 upgrade.
+	UpgradeNames = []string{"v1.0.0", "v1.0.5", "v1.1.0"}
+	// UpgradeHeight defines the set of block heights at which the classic terra v1 upgrade is
+	// triggered.
+	UpgradeHeights = []int64{core.SwapDisableForkHeight, core.SwapEnableForkHeight, core.VersionMapEnableHeight}
```

Comment:
@edk208 @inon-man Do you know which versions are used for these three fork upgrades? I can't find some historical versions in this repo.

Reply:
> I think we can delete old soft-forks too, because it needs to be run with v1.0.5 anyway

@inon-man You mean classic terra reboot mainnet from genesis with v1.0.5?

Reply:
> It's all over the place

Oh, looks a bit confused.